katsdpingest/ingest_session.py

```diff
@@ -185,6 +189,26 @@ def _fix_descriptions(desc: Any) -> Any:
         return desc
 
 
+def _load_model(telstate: katsdptelstate.TelescopeState,
```

Comment:
This seems like the kind of thing we would repeat in every part of the system that wants to use models - could this not be in the models library or telstate ?

Reply:
Good point, we definitely need somewhere for it. I felt reluctant about adding a new dependency of katsdpmodels, but I guess it could be an optional dependency. I'll have a go at it.